alchemy/src/vercel/storage.ts

```diff
@@ -0,0 +1,184 @@
+import type { Context } from "../context.ts";
+import { Resource } from "../resource.ts";
+import type { Secret } from "../secret.ts";
+import { createVercelApi, VercelApi } from "./api";
+
+type StorageType = "blob";
+
+/**
+ * Properties for creating or updating a Storage
+ */
+export interface StorageProps {
+  /**
+   * The desired name for the storage
+   */
+  name: string;
+
+  /**
+   * The region where the storage will be deployed
+   */
+  region: string;
+
+  /**
+   * The team ID that the storage belongs to
+   */
+  teamId: string;
```

Comment:
Do you think we'll eventually have a `Team` resource? If yes, then this should be `team: string` so that it can be later `team: string | Team` which is our convention for supporting both identifiers and references to alchemy resources.